visualization/app/codeCharta/ui/copyToClipboardButton/copyToClipboardButton.component.ts

```diff
@@ -0,0 +1,15 @@
+import "./copyToClipboardButton.component.scss"
+import { Component, Inject } from "@angular/core"
+import { CopyToClipboardService } from "./copyToClipboard.service"
+
+@Component({
+	selector: "cc-copy-to-clipboard-button",
+	template: require("./copyToClipboardButton.component.html")
+})
+export class CopyToClipboardButtonComponent {
+	constructor(@Inject(CopyToClipboardService) private service: CopyToClipboardService) {}
```

Comment:
[Optional] rename service -> copyToClipboardService or copyToClipboard

Reply:
Sry, but I do not get the benefit from this renaming. The name does not need to be the same as the type, this is what your code editor is for.
Of course, if there are several services in this component, then this would be a different situation...